### PR TITLE
Drawer: Add visibility: hidden when drawer panel is fully closed

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: packages/devextreme/js/ui/drawer/ui.drawer.js [hunk 1]
```diff
@@ -194,16 +195,37 @@ const Drawer = Widget.inherit({
         }
     },
 
+    _updateHasMinSizePanelContentClass(shouldBeSet) {
+        if(shouldBeSet) {
+            this._$panelContentWrapper.addClass(DRAWER_PANEL_CONTENT_HAS_MIN_SIZE_CLASS);
+        } else {
+            this._$panelContentWrapper.removeClass(DRAWER_PANEL_CONTENT_HAS_MIN_SIZE_CLASS);
```
Comment (ksercs) [inline packages/devextreme/js/ui/drawer/ui.drawer.js]: We can use `toggle` method here to make the solution more compact

File: packages/devextreme/js/ui/drawer/ui.drawer.js [hunk 2]
```diff
@@ -194,16 +195,37 @@ const Drawer = Widget.inherit({
         }
     },
 
+    _updateHasMinSizePanelContentClass(shouldBeSet) {
```
Comment (ksercs) [inline packages/devextreme/js/ui/drawer/ui.drawer.js]:
```suggestion
    _updatePanelContentHasMinSizeClass(shouldBeSet) {
```

File: packages/devextreme/js/ui/drawer/ui.drawer.js (hunk 1 above)
Comment (marker-dao) [reply packages/devextreme/js/ui/drawer/ui.drawer.js]: [bb69e04e6cd524f3cf7095ba2d578a1f830db4d7]

File: packages/devextreme/js/ui/drawer/ui.drawer.js (hunk 2 above)
Comment (marker-dao) [reply packages/devextreme/js/ui/drawer/ui.drawer.js]: [bb69e04e6cd524f3cf7095ba2d578a1f830db4d7]

**Review by ksercs — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by iBat — APPROVED** [review]

**Pull request merged**